### PR TITLE
Rightsizing

### PR DESCRIPTION
**Pull request by jackzampolin**
Fixes https://github.com/influxdb/influxdb.com/issues/188 and #205

### CODE REVIEW

File: content/docs/v0.9/guides/hardware_sizing.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Hardware Sizing Guidelines
+aliases:
+  - /docs/v0.9/guides/hardware_sizing.html
+---
+
+InfluxDB is a high performance database optimized for time series, but like any software its performance is affected by the hardware running it. Below are some recommendations for how to optimize your InfluxDB instance and understand which performance limitations are hardware related and which are software related. Below is a general list of hardware recommendations from the InfluxDB team.
+
+| Load         | Writes/Second  | Queries/Second | Unique Series |
```
Comment (beckettsean) [inline content/docs/v0.9/guides/hardware_sizing.md]: I pulled these numbers from thin air when I wrote them down. Do we have data to back up these numbers? Can you start an email thread with performance-minded folks to get better consensus numbers for this table?

**Comment by beckettsean** [comment]
@jackzampolin I had intended my doc as a start for the page, not the entire contents. Are you planning to add more details and make these into complete sentences, etc? If not I can take a look later, but this is un-releaseable as is.

**Comment by jackzampolin** [comment]
@beckettsean Will do

**Comment by beckettsean** [comment]
@jackzampolin can this be ported to docs.influxdata.com repo?

**Comment by rkuchan** [comment]
Closed in favor of: https://github.com/influxdb/docs.influxdata.com/pull/92

**Pull request closed without merging**